Nuget/Ploeh.AutoFixture.nuspec

```diff
@@ -6,6 +6,9 @@
         <title>AutoFixture</title>
         <authors>Mark Seemann</authors>
         <owners>Mark Seemann</owners>
+        <dependencies>
+            <dependency id="Albedo" version="[1,2)" />

```

Comment:
Is this the correct dependency configuration? The actual dependency added is Albedo 1.0.1, but if I understand this configuration correctly, any 1.x version is supported - including 1.0.0.

Additionally, why exclude a hypothetical version 2 of Albedo?